mhs/mhs/outbound/request/synchronous/handler.py

```diff
@@ -1,91 +1,108 @@
 """This module defines the outbound synchronous request handler component."""
 
-import datetime
-from typing import Dict, Callable
+from typing import Dict
 
 import tornado.locks
 import tornado.web
 
 import mhs.common.workflow.common as common_workflow
-import mhs.common.workflow.sync_async as sync_async_workflow
 import mhs.outbound.request.common as common
-from utilities import integration_adaptors_logger as log
+from mhs.common import workflow
+from mhs.common.configuration import configuration_manager
+from utilities import integration_adaptors_logger as log, message_utilities
 
 logger = log.IntegrationAdaptorsLogger('MHS_OUTBOUND_HANDLER')
 
 
 class SynchronousHandler(common.CommonOutbound, tornado.web.RequestHandler):
     """A Tornado request handler intended to handle incoming HTTP requests from a supplier system."""
 
-    def initialize(self, workflow: sync_async_workflow.SyncAsyncWorkflow,
-                   callbacks: Dict[str, Callable[[str], None]], async_timeout: int):
+    def initialize(self, workflows: Dict[str, workflow.CommonWorkflow],
+                   config_manager: configuration_manager.ConfigurationManager):
         """Initialise this request handler with the provided configuration values.
 
-        :param workflow: The workflow to use to send messages.
-        :param callbacks: The dictionary of callbacks to use when awaiting an asynchronous response.
-        :param async_timeout: The amount of time (in seconds) to wait for an asynchronous response.
+        :param workflows: The workflows to use to send messages.
+        :param config_manager: The object that can be used to obtain configuration details.
         """
-        self.workflow = workflow
-        self.callbacks = callbacks
-        self.async_response_received = tornado.locks.Event()
-        self.async_timeout = async_timeout
+        self.workflows = workflows
+        self.config_manager = config_manager
 
-    async def post(self, interaction_name):
-        logger.info('0001', 'Client POST received. {Request}', {'Request': str(self.request)})
+    async def post(self):
+        message_id = self._extract_message_id()
+        self._extract_correlation_id()
 
-        message_id = self.get_query_argument("messageId", default=None)
+        logger.info('0006', 'Client POST received. {Request}', {'Request': str(self.request)})
```

Comment:
Pretty minor but just thinking for consistency

```suggestion
        logger.info('0006', 'Outbound POST received. {Request}', {'Request': str(self.request)})
```